### PR TITLE
Flat Icons for all the events.

### PR DESCRIPTION
**Pull request by PawanKolhe**
These are the logos. I still have improvements to make, such as the 'ACM Fair icon' may need to be changed. I would like some feedback.

### CODE REVIEW

**Comment by yvtheja** [comment]
Change the ACM fair and Geekend icons.

**Comment by PawanKolhe** [comment]
ok.

**Comment by yvtheja** [comment]
And the pull request is showing 3 commits !! I need PR to contain only one commit.

**Comment by PawanKolhe** [comment]
how do i do tht?

File: index.html [hunk 1]
```diff
@@ -172,7 +172,7 @@ <h2 class="wow fadeInLeftBig">Our Services</h2>
                   <!-- BEGIN SINGLE SERVICE -->
                   <div class="single_service wow fadeInLeft">
                     <div class="service_iconarea">
-                      <span class="fa fa-paper-plane service_icon"></span>
+                      <img src="img/services/InOut.png" height="150px">
```
Comment (yvtheja) [inline index.html:175]: @PawanKolhe, you shouldn't use the inline styling ! You should make the icons dimensions to 128px or 150px !

File: index.html [hunk 2]
```diff
@@ -182,7 +182,7 @@ <h3 class="service_title">HACKATHON - InOut</h3>
                   <!-- BEGIN SINGLE SERVICE -->
                   <div class="single_service wow fadeInRight">
                     <div class="service_iconarea">
-                      <span class="fa fa-suitcase service_icon"></span>
+                      <img src="img/services/Epiphany.png" height="150px">
```
Comment (yvtheja) [inline index.html:185]: same as above

File: index.html [hunk 3]
```diff
@@ -192,9 +192,9 @@ <h3 class="service_title">Epiphany Series</h3>
                   <!-- BEGIN SINGLE SERVICE -->
                   <div class="single_service wow fadeInLeft">
                     <div class="service_iconarea">
-                      <span class="fa fa-paper-plane service_icon"></span>
+                      <img src="img/services/Tech.png" height="150px">
```
Comment (yvtheja) [inline index.html:195]: same as above

File: index.html [hunk 4]
```diff
@@ -202,7 +202,7 @@ <h3 class="service_title">TECHIQ-2.0 & 1.0</h3>
                   <!-- BEGIN SINGLE SERVICE -->
                   <div class="single_service wow fadeInRight">
                     <div class="service_iconarea">
-                      <span class="fa fa-paper-plane service_icon"></span>
+                      <img src="img/services/Workshop.png" height="150px">
```
Comment (yvtheja) [inline index.html:205]: same as above

File: index.html [hunk 5]
```diff
@@ -212,7 +212,7 @@ <h3 class="service_title">Basic Workshops</h3>
                   <!-- BEGIN SINGLE SERVICE -->
                   <div class="single_service wow fadeInLeft">
                    <div class="service_iconarea">
-                      <span class="fa fa-envelope-o service_icon"></span>
+                      <img src="img/services/Geek.png" height="150px">
```
Comment (yvtheja) [inline index.html:215]: same as above

File: index.html [hunk 6]
```diff
@@ -222,7 +222,7 @@ <h3 class="service_title">ACM Geekend</h3>
                   <!-- BEGIN SINGLE SERVICE -->
                   <div class="single_service wow fadeInRight">
                    <div class="service_iconarea">
-                      <span class="fa fa-envelope-o service_icon"></span>
+                      <img src="img/services/HourOfCode.png" height="150px">
```
Comment (yvtheja) [inline index.html:225]: same as above

File: index.html [hunk 7]
```diff
@@ -233,7 +233,7 @@ <h3 class="service_title">Hour of Code</h3>
                   <!-- BEGIN SINGLE SERVICE -->
                   <div class="single_service wow fadeInLeft">
                    <div class="service_iconarea">
-                      <span class="fa fa-envelope-o service_icon"></span>
+                      <img src="img/services/ACMFair.png" height="150px">
```
Comment (yvtheja) [inline index.html:236]: same as above

File: index.html [hunk 8]
```diff
@@ -243,7 +243,7 @@ <h3 class="service_title">ACM Fair</h3>
                   <!-- BEGIN SINGLE SERVICE -->
                   <div class="single_service wow fadeInRight">
                    <div class="service_iconarea">
-                      <span class="fa fa-envelope-o service_icon"></span>
+                      <img src="img/services/ACMQuest.png" height="150px">
```
Comment (yvtheja) [inline index.html:246]: same as above ! And try to change the icon for Quest also :)

File: index.html [hunk 9]
```diff
@@ -192,9 +192,9 @@ <h3 class="service_title">Epiphany Series</h3>
                   <!-- BEGIN SINGLE SERVICE -->
                   <div class="single_service wow fadeInLeft">
                     <div class="service_iconarea">
-                      <span class="fa fa-paper-plane service_icon"></span>
+                      <img src="img/services/Tech.png" height="150px">
                     </div>
-                    <h3 class="service_title">TECHIQ-2.0 & 1.0</h3>
+                    <h3 class="service_title">TECHIQ - 2.0 & 1.0</h3>
```
Comment (yvtheja) [inline index.html:197]: @PawanKolhe , why you gave spaces in between ??

**Comment by yvtheja** [comment]
@PawanKolhe , nice work ! apply the above mentioned changes and merge the commits into one ! Try searching Internet for it or you can ping me !

**Pull request closed without merging**